builtin/providers/aws/resource_aws_iam_user.go

```diff
@@ -173,6 +184,16 @@ func resourceAwsIamUserDelete(d *schema.ResourceData, meta interface{}) error {
 		}
 	}
 
+	for _, k := range accessKeys {
```

Comment:
Would you mind moving this for loop along with `var accessKeys []string` into the `force_destroy` condition too?
It IMO makes sense from the context point of view.